code/addons/vitest/src/updateVitestFile.ts

```
@@ -11,7 +11,10 @@ export const loadTemplate = async (name: string, replacements: Record<string, st
     join(resolvePackageDir('@storybook/addon-vitest'), 'templates', name),
     'utf8'
   );
-  Object.entries(replacements).forEach(([key, value]) => (template = template.replace(key, value)));
+  // Normalize Windows paths (backslashes) to forward slashes for JavaScript string compatibility
+  Object.entries(replacements).forEach(
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)